cmd/kubeadm/app/node/bootstrap.go

```diff
@@ -101,15 +91,16 @@ func EstablishMasterConnection(c *kubeadmapi.TokenDiscovery, clusterInfo *kubead
 		close(result)
```

Comment:
wg.Wait() should not be defered here, it should be synchronous and the close(result) is unnecessary and should be elided